spec/models/fly_spec.rb

```diff
@@ -5,10 +5,10 @@
     it { should validate_presence_of(:name) }
     it { should validate_presence_of(:size) }
     it { should validate_presence_of(:color) }
-    it { should validate_presence_of(:type) }
+    it { should validate_presence_of(:category) }
   end
 
-  describe 'relationships' do
-    it { should belong_to(:user) }
-  end
+  # describe 'relationships' do
+  #   it { should belong_to(:user) }
+  # end
```

Comment:
Tech debt here to discuss.. No users so the users flybox index will just be the fly index action.

Reply:
Yes, we are focusing for core fly and fish functionality for now, will go back and add user functionality later